src/topp/TextExporter.cpp

```diff
@@ -213,6 +235,14 @@ namespace OpenMS
     {
       out << str + suffix;
     }
+    if (incl_IIMN & write_IIMN)
```

Comment:
why are there two variables needed?
what does incl_IIMN mean here?
please don't use & but &&

Reply:
Right, the two variables are not needed.